server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -601,6 +611,15 @@ public FlowInstanceDetailResp approve(@NotNull Long id, String message, Boolean
             return response.getContentByType(new TypeReference<SuccessResponse<FlowInstanceDetailResp>>() {}).getData();
         }
         completeApprovalInstance(id, instance -> instance.approve(message, !skipAuth), skipAuth);
+        if (notificationProperties.isEnabled()) {
+            try {
+                Event event =
```

Comment:
There is no special reason. When writing the design document, only some events that users may be interested in were selected, rather than incorporating all events into message notifications. And we also referred to products such as DMS and Ninedata, and they did not have any `cancel` events. So we didn't support this event either 😂